apis/placement/v1alpha1/disruptionbudget_types.go

```diff
@@ -0,0 +1,149 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster,categories={fleet,fleet-placement},shortName=crpdb
+// +kubebuilder:subresource:status
+// +kubebuilder:storageversion
+
+// ClusterResourcePlacementDisruptionBudget is the policy applied to a ClusterResourcePlacement
+// object that specifies its disruption budget, i.e., how many placements (clusters) can be
+// down at the same time due to **voluntary** disruptions (e.g., evictions).
+//
+// To apply a ClusterResourcePlacementDisruptionBudget to a ClusterResourcePlacement, use the
+// same name for the ClusterResourcePlacementDisruptionBudget object as the ClusterResourcePlacement
+// object. This guarantees a 1:1 link between the two objects.
+type ClusterResourcePlacementDisruptionBudget struct {
```

Comment:
type PlacementDisruptionBudget struct {

How about

Reply:
Yeah, will update the name.